src/pudl/metadata/resources/ferc1.py

```diff
@@ -823,6 +823,22 @@
         "etl_group": "ferc1",
         "field_namespace": "ferc1",
     },
+    "denorm_plants_utils_ferc1": {
```

Comment:
We should standardize on using either the full "utilities" or the abbreviation "utils" a mix will be confusing.

Reply:
Agreed. I say we go with the more verbose options for naming tables.